README.md

```diff
@@ -108,7 +108,82 @@ Known limitations at this point include but are not limited to:
 
 ## Bugs
 
-
 If you are having issues, we would love to hear from you as we work to make this product better:
 
 https://github.com/pulumi/pulumi-terraform-module/issues
+
+## Module configuration and schema override
```

Comment:
Very nice! I'm itching to iterate a bit on how we present this feature in the README but it's probably best done in subsequent PRs.